src/components/Article.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticle } from "../utils/api";
+import Articles from "./Articles";
+
+function Article() {
+  const [currArticle, setArticle] = useState({ created_at: "0000-00-00" });
+
+  const [isLoading, setIsLoading] = useState(true);
+
+  const [isError, setError] = useState(null);
+
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getArticle(article_id)
+      .then((articleFromApi) => {
+        setArticle(articleFromApi.article);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        setError(err.response.data);
+      });
+  }, [article_id]);
+
+  if (isError) {
+    return <p className="error">Oopps.. some error occured!!</p>;
```

Comment:
Could you make this error message a bit more specific e.g. Error 404 - that article does not exist/not found?